### ISSUE TITLE
404 when saving post

### ISSUE THREAD

**Issue by joefatora** [netlify/netlify-cms#393]
**- Do you want to request a *feature* or report a *bug*?**
Bug

**- What is the current behavior?**
When I try to save a post, I get a warning saying `Failed to persist entry: API_ERROR: Not Found`
and the github API returns a 404.

**- If the current behavior is a bug, please provide the steps to reproduce.**
using this basic config with a metalsmith site:

```yaml
backend:
  name: github
  repo: bequest/external-site
  branch: master

public_folder: "dist/images/uploads"
media_folder: "src/images/uploads"

collections:
  - name: "blog" # Used in routes, e.g. /admin/collections/blog
    label: "Blog"
    folder: "src/_posts/blog" # The path to the folder where the documents are stored
    create: true # Allow users to create new documents in this collection
    slug: "{{year}}-{{month}}-{{day}}-{{slug}}" # Filename template i.e. YYYY-MM-DD-title.md
    fields: # The fields for each document, usually in front matter
      - {label: "Layout", name: "layout", widget: "hidden", default: "blog"}
      - {label: "Title", name: "title", widget: "string"}
      - {label: "Publish Date", name: "date", widget: "datetime"}
      - {label: "Featured Image", name: "thumbnail", widget: "image"}
      - {label: "Body", name: "body", widget: "markdown"}
```

I am able to access the CMS at the netlify url, but cannot save after editing a post
**- What is the expected behavior?**
Post saves without error.

**- Please mention your node.js, and operating system version.**
Using netlify to deploy. No node version is set in package.json

**Comment by almeynman** [comment]
I have similar problem
![image](https://cloud.githubusercontent.com/assets/6459157/26580191/e70d311c-4558-11e7-880b-3027bbe6f50b.png)

**Comment by almeynman** [comment]
I was having this problem with editorial_workflow enabled. After disabling it, save button still won't work, without any errors in the console.


I have deleted OAuth Application from GitHub and deleted Github Provider. After resetting these, I was able to see some activity when clicking Save button.


FIrst of all These errors appears without any actions from my side. 
![image](https://cloud.githubusercontent.com/assets/6459157/26584338/4831ad0e-456a-11e7-8088-7201d9821d69.png)


When Clicking save button this happens:
![image](https://cloud.githubusercontent.com/assets/6459157/26584466/aa8bfc66-456a-11e7-9899-f7ab5c764c52.png)

**Comment by erquhart** [comment]
@joefatora we never did follow up on this ticket - are you still having issues?

I've helped a number of folks with what you're seeing (including @almeynman), and so far it's always an authentication issue. The following causes have surfaced:

1. Running locally as 127.0.0.1 instead of localhost
2. Cookie problems (clear all site data for the domain where you're  having issues and reload)
3. Using an old version of the CMS
4. Misspelling the repo information in the config
5. Not having an oauth provider setup (eg. Netlify) when using the CMS on a live site (not running locally)

For anyone seeing this issue, visit us on [Gitter](https://gitter.im/netlify/NetlifyCMS) for help.

**Comment by luckyrat** [comment]
Here's another authentication issue for the list that causes this 404 error.

I created an OAuth application on my GitHub user account but couldn't get netlify CMS to write to the repo until I deleted that and instead created the OAuth application on the settings page for the organisation that contains the content repo. The user logging in to netlify CMS is a Write collaborator on only the one repo that contains the website code and content.

Perhaps this is just a detail of GitHub OAuth that everyone but me already knows but it was a bit confusing and no docs seem to mention this tip for those of us new to this method of authentication within GitHub so I thought it worth mentioning here.
